web/src/games/checkers/game.test.ts

```diff
@@ -15,8 +18,21 @@ test('invalid moves', () => {
 });
 
 it('should declare player 1 as the winner', () => {
+  const CheckersGameWithSetup = {
+    ...CheckersGame,
+    setup: () => ({
+      board: INITIAL_BOARD[0],
+      jumping: null,
+      config: {
+        ...DEFAULT_FULL_CUSTOMIZATION,
+        flyingKings: true,
+        stopJumpOnKing: false,
```

Comment:
Ideally we would add a bit more of coverage here, for these new configs.

Reply:
I agree - the only reason I didn't is that these config options are the bare minimum to allow the already written full game test to work. I'd really prefer not to rewrite every move by hand all over again with the new configs, so maybe a better way would be more general unit tests?

Reply:
Oh, I think that maybe playing a single game for each config would be enough. No need to rewrite everything. We can merge this PR and follow up later. Everything LGTM.